People/Patient.cs

```diff
@@ -0,0 +1,99 @@
+﻿namespace People
+{
+    public class Patient : Person
+    {
+        private bool isSick;
```

Comment:
Missing underscore for private fields Names

Reply:
Oops, fixed it.